src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -197,6 +199,7 @@ public LivingEntity(ModElement element) {
 
 		this.entityDataEntries = new ArrayList<>();
 		this.modelLayers = new ArrayList<>();
+		this.spawnPercentageRaid = new int[] {0, 0, 0, 0, 0, 0, 0};

```

Comment:
I asked you to match default values here with default values in UI, but this did not happen yet: 

![image](https://github.com/MCreator/MCreator/assets/16374228/1ea9ca11-f57a-43f8-a787-ef76abd18b43)

Reply:
Done!